lib/Minz/Translate.php

```diff
@@ -79,6 +81,31 @@ public static function availableLanguages() {
 		return array_unique($list_langs);
 	}
 
+	/**
+	 * Return the language to use in the application.
+	 * It returns the connected language if it exists then returns the first match from the
+	 * preferred languages then returns the default language
+	 * @param $user the connected user language (nullable)
+	 * @param $preferred an array of the preferred languages
+	 * @param $default the preferred language to use
+	 * @return a string containing the language to use
+	 */
+	public static function getLanguage($user, $preferred, $default) {
+		if (null !== $user) {
+			return $user;
+		}
+
+		$languages = Minz_Translate::availableLanguages();
+		foreach ($preferred as $language) {
+			$language = strtolower($language);
+			if (in_array($language, $languages)) {
```

Comment:
```suggestion
			if (in_array($language, $languages, true)) {
```

I would still feel safer with a strict type comparison here https://php.net/in-array

Reply:
done